docs/uglify-options.md

```diff
@@ -7,113 +7,94 @@ This task primarily delegates to [UglifyJS2][], so please consider the [UglifyJS
 
 
 ## mangle
-Type: `Boolean` `Object`  
```

Comment:
No worries. BTW, you can skip README.md in your patch; it will get updated automatically with the next version bump.